lib/generators/rolemodel/css/base/base_generator.rb

```diff
@@ -6,6 +6,10 @@ class BaseGenerator < Rails::Generators::Base
       include Rolemodel::BundlerHelpers
       source_root File.expand_path('templates', __dir__)
 
+      def install_webpack
+        generate 'rolemodel:webpacker'
```

Comment:
Not sure I'd add this here. Is something now dependent on that?

Reply:
I figured the webpacker generator was required for css base since all the styles are served through webpacker. This way, icons depends on css, and css depends on webpacker. Do you see a scenario where the css generator would be run without running the webpacker one?

Reply:
I don't see a case where it won't be run. It just probably has already been run by the time we get to this point. Don't really need/want to run it again in all cases